### PR TITLE
added singularity.autoMounts to config

### PR DESCRIPTION
**Pull request by marcodelapierre**
otherwise a run with -with-singularity will fail, not being able to find the transcriptome file

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -22,12 +22,14 @@ manifest {
 profiles {
   standard {
     process.container = 'nextflow/rnaseq-nf:latest'
+    singularity.autoMounts = true
   }
```
Comment (pditommaso) [inline nextflow.config]: Let's add it only in the slurm profile, it can be confusing to have it here.

**Comment by marcodelapierre** [comment]
My only concern is that this pipeline can easily be used for demo/training purposes, as it's small enough. In these contexts, one might end up running it with singularity but without Slurm (e.g. laptop or VM). If it's confusing, a comment can be added for that line, e.g.
```
// the line below is required for singularity to mount all input paths, including for transcriptome file
singularity.autoMounts = true
```
What do you think?
Thanks

**Comment by pditommaso** [comment]
What if we add `singularity` profile specific for that? eg

```
singularity {
  singularity.autoMounts = true
  singularity.enabled = true
}

**Comment by marcodelapierre** [comment]
cool! just modified the config

**Comment by marcodelapierre** [comment]
thanks Paolo!

and just to let you know, Nextflow has now got its own independent episode in our Singularity tutorial ;-)
I will use these materials as early as this December in Sydney!

Cheers,
Marco

**Pull request closed without merging**